elastic/elastic-agent
# Fix creatork8sscript

- Bug

## What does this PR do?

Fixes a bug for wrong PRs that the automation was creating like the https://github.com/elastic/kibana/pull/165606/files

The [creator_k8s_manifest](https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/creator_k8s_manifest.sh) 
was inlcuding the ksm configmaps wrongly in the ts files of kibana PRs

## Why is it important?

Fixes the false automated PRs that can create wrong manifests in Kibana

## Checklist





## How to test this PR locally

```bash
cd deploy/kubernetes
WITHOUTCONFIG=true make generate-k8s
 ./creator_k8s_manifest.sh .
```

Open elastic_agent_manifest.ts and check diffs with https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/services/elastic_agent_manifest.ts

## Use cases

(https://github.com/elastic/kibana/pull/165606/files)

## Screenshots

Diffs that we dont want to see anymore

![Screenshot 2023-09-11 at 4 00 41 PM](https://github.com/elastic/elastic-agent/assets/1685415/db3fb145-51d3-451b-83ba-8c853ab065e6)
